src/test/java/com/cloudera/sparkts/api/java/JavaTimeSeriesRDDFactoryTest.java

```diff
@@ -258,81 +249,71 @@ public void testSaveLoad() {
 
     @Test
     public void testToIndexedRowMatrix() {
-        JavaSparkContext sc = init();
-
-        ZonedDateTime start = ZonedDateTime.of(2015, 4, 9, 0, 0, 0, 0, ZoneId.systemDefault());
-        String[] labels = new String[]{ "a", "b", "c", "d", "e" };
-        double[] seeds = untilBy(0, 20, 4);
-        List<Tuple2<String, Vector>> list = new ArrayList<>();
-        for(int i = 0; i < seeds.length; i++) {
-            double seed = seeds[i];
-            list.add(new Tuple2<>(labels[i],
-                    (Vector) new DenseVector(until((int) seed, (int) seed + 4))));
-        }
-        UniformDateTimeIndex index = DateTimeIndexFactory.uniform(start, 4, new DayFrequency(1));
+        try (JavaSparkContext sc = init()) {
```

Comment:
It's to auto-close it, as if in a finally block. I suppose this makes sure that one failed test does gum up the others with a live additional context?